app/helpers/log_list_helper.rb

```diff
@@ -0,0 +1,25 @@
+module LogListHelper
+  def display_delete_logs?(current_user, search_term)
+    if current_user.data_provider?
+      filter_selected?("user", "yours")
+    else
+      any_filter_selected? || search_term.present?
+    end
+  end
+
+  def delete_logs_path(logs, search_term)
+    controller_type = controller.class.name.delete_suffix("Controller").underscore.to_sym
```

Comment:
We shouldn't have to do this. Can you explain what you're trying to din this method?

Reply:
I need to insert a Delete Logs button into the index page (_log_list partial) which is reused for all different log types and can be rendered by three different controllers.
I'm up for discussing this and further thoughts on it, but I also think the implementation in anger of this functionality can be worried about when I'm implementing for these other controllers.

Reply:
🤦‍♂️